src/Nethermind/Nethermind.TxPool.Test/TxBroadcasterTests.cs

```diff
@@ -265,7 +265,10 @@ public void should_skip_blob_txs_when_picking_best_persistent_txs_to_broadcast([
                 .WithShardBlobTxTypeAndFieldsIfBlobTx()
                 .SignedAndResolved(_ethereumEcdsa, TestItem.PrivateKeys[i])
                 .TestObject;
+        });
 
+        for (int i = 0; i < addedTxsCount; i++)
+        {
```

Comment:
Broadcast should be thread safe so you can put it under `Parallel.For`

Reply:
At least it should be - order doesn't matter here so worth to move to parallelized part and verify it